front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -152,6 +152,45 @@
         word-break: break-all;
       }
 
+      .not-opened {
+        position: relative;
```

Comment:
I think the `position: relative` isn't necessary anymore.

Reply:
Done